chess/db/package.json

```diff
@@ -6,7 +6,8 @@
   "types": "build/index.d.ts",
   "type": "module",
   "scripts": {
-    "build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json",
+    "clean": "rm -rf tsconfig.tsbuildinfo & rm -rf build",
```

Comment:
can you change this to use the `wipe.sh` file instead?

Reply:
done!